src/main/resources/default/assets/common/scripts/continuity.js.pasta

```diff
@@ -424,5 +430,33 @@ window.sirius.Continuity = (function () {
         }
     }
 
+    function deepEqual(object1, object2) {
```

Comment:
super optional - we migth make that a public sirius method (?) might be re-usable (??)